bull-map-transformer/src/test/java/com/hotels/map/transformer/MapTransformerTest.java

```diff
@@ -214,34 +206,32 @@ public void testTransformWorksProperlyWithTargetKeyAndElemType() {
         Map<MutableToFooSimple, Map> actual = underTest.transform(EXTREME_COMPLEX_MAP, MutableToFooSimple.class, Map.class);
 
         //THEN
-        assertNotNull(actual);
-        assertEquals(actual.size(), EXTREME_COMPLEX_MAP.size());
+        assertThat(actual).isNotNull();
+        assertThat(actual.size()).isEqualTo(EXTREME_COMPLEX_MAP.size());
         // check that the element has been converted
         for (Map.Entry<MutableToFooSimple, Map> entry : actual.entrySet()) {
-            assertEquals(MutableToFooSimple.class, entry.getKey().getClass());
+            assertThat(entry.getKey().getClass())
+                    .isEqualTo(MutableToFooSimple.class);
         }
```

Comment:
Did you consider applying the simplification I proposed?
https://github.com/HotelsDotCom/bull/pull/161#discussion_r445456960

Reply:
I missed it, thank you